CHANGELOG.md

```diff
@@ -80,6 +80,7 @@ Consider migrating to `stream-chat-android-ui-components` or `stream-chat-androi
 ### ⬆️ Improved
```

Comment:
Maybe add a note that an animation for `ChannelInfo` was added to `ChannelScreen`.

Seems like it would improve user experience so our users should know about it.